setup.py

```diff
@@ -11,7 +11,7 @@
 
 setup(
     name='auditor',
-    version='1.3.0',
+    version='1.4.0',
```

Comment:
**suggestion**: bumping versions in branches can get a little funky when features etc happen asynchronously or on top of each other. It is simpler to bump versions only on master.

Reply:
Yeah, I've been seeing how that can get tricky, including having the version increment buried in a random commit. Is the best procedure to just do all the code updates for the new feature, do the PR, and then after the PR has been merged go do a single commit to increment the version right on master w/o a PR review?

Reply:
you got it.